data-lineage/src/main/java/com/wipro/ats/bdre/lineage/entiity/Function.java

```diff
@@ -47,6 +47,11 @@ public Function(String label, List<String> inColumnNameList, String outAlias) {
 //		++counter;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)